frontend/src/scenes/events/eventsTableLogic.js

```diff
@@ -48,7 +48,12 @@ export const eventsTableLogic = kea({
         props.key,
 
     actions: () => ({
-        setProperties: (properties) => ({ properties }),
+        setProperties: (properties) => {
+            if (properties === []) {
+                return { properties: {} }
+            }
+            return { properties }
+        },
```

Comment:
This will always be false (reference comparison with a new array `[]`).

Reply:
Also, the cleaner way to write this is something like:
```
setProperties: (properties) => ({ properties: properties || {} })
```

Reply:
wild, you're right of course. It avoids the bug where we can't load events when properties is an empty array 🤯  

I'd propose we merge as is. Then hopefully this code disappears in the conversion to TS

Reply:
Switched to a not silly comparison and that also works as a workaround... and won't surprise anyone in the meanwhile :)